### DISCUSSION TITLE
Yakit建议

### DISCUSSION THREAD

**Discussion by ghost** [yaklang/yakit#84] [General]
Yakit的code界面添加一些报错功能，如语法错误报红、参数类型错误报红、拼写错误报红等功能。对用户体验的角度会更友好。

**Comment by VillanCh** [comment]
感谢大哥建议

你说的对，现在只加了自动补全，lint 和 autoFix 正在研究，有🤏一点点困难，正在研究中，基本和 vscode 插件编写类似，但是实际语法检查编写更复杂一些，应该不久就可以解决了

**Comment by Vernon818** [comment]
@L-tgdd 为了感谢您对yak和yakit的支持，我们想邮寄一份小礼物给您，麻烦您微信添加一下：Juzi19880818，并备注好您的github账号。如已经添加账号，请微信联系我一下，谢谢

**Comment by 989800039** [comment]
使用困难，都是国产的为什么不搭配通义千问呢？使用的时候很重，web初级根本看不懂，用这个还得学一个新的语言？似乎太重了啊
